CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed default distributed training strategy from single-GPU to FSDP
 - Fixed behavior of `effective_memmap_dtype` to prevent unrecognized dtypes to be parsed as `uint16`. 
 
+### Fixed
+
+- Fixed restarting a training run in later epochs so that we no longer need to set the flag `--epoch=INT`. 
```

Comment:
Should we remove the `--epoch` flag?

Reply:
It might be useful at some point so I'd rather keep it for now